dilivery.html

```diff
@@ -5,7 +5,7 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
You should have changed this instead
```suggestion
    <meta name="viewport1234213412" content="width=device-width, initial-scale=1.0">
```